### DISCUSSION TITLE
📢 Important Announcement: README Update!

### DISCUSSION THREAD

**Discussion by mihir-bombay-studio** [dear-digital/linter#51] [Announcements]
Hello Team! 👋

We've just rolled out some crucial updates to the README file! 🎉 The updated README will be effective from September 1, 2023, so please make sure to read it thoroughly to get up-to-date on the new changes. Here's a quick overview:

1️⃣ **Internship Overview (Updated)**  
We have introduced new guidelines for the internship, including milestone-based approaches and weekly commitments.

2️⃣ **Points System (Updated)**  
No more self-created issues! All issues will now be created by the project lead. Also, there are some new rules about how points are assigned and calculated.

3️⃣ **Mentorship and Feedback Sessions (Updated)**  
Live feedback sessions have been discontinued. All discussions will now be carried out in the discussions channel.

4️⃣ **Code of Conduct & FAQ (Updated)**  
Make sure to familiarize yourself with the updated Code of Conduct and FAQs for better collaboration and a more informed experience.

5️⃣ **Current Project Phase**  
The current focus is on refactoring and adding coding style rules.

To be in sync with the project's latest guidelines, please make sure you go through the entire [README](https://github.com/dear-digital/linter/blob/bdd6b006b983e9ef93c2a1417dc801e5b5f4bdd3/README.md) file.

Thank you for your attention, and let's keep pushing the boundaries of what we can achieve together! 🚀

**Comment by tanujbordikar** [comment]
Hi @mihir-bombay-studio Sir, I am Tanuj Bordikar and i want to contribute in this repository and earn 100 points to work as an Intern! So how can I earn 100 points?

**Reply by mihir-bombay-studio** [reply]
Hi @tanujbordikar, There must be 1 active issue with the label "open task" that you can start with even if it has a submission already present you can add your own submission there. I'll add new open tasks as well later today so you can start earning points

**Comment by tanujbordikar** [comment]
Okay I will work on that task, sir.
